# Define the goal
string position
---
# Define the result
bool succeed
---
# Define a feedback message
float32 percent_complete